build-test-cxx.yml

```diff
@@ -0,0 +1,124 @@
+name: 'Build, test, package'
+description: 'Build, test, and package distributions for an ITK external module.'
+inputs:
+  c-compiler:
+    required: true
+  cxx-compiler:
+    required: true
+  cmake-build-type:
+    required: false
+    default: 'MinSizeRel'
+  cmake-options:
+    description: 'Additional CMake arguments to use in building and testing the current remote module.'
+    required: false
+  itk-cmake-options:
+    description: 'Additional CMake arguments to use in building ITK, such as enabling certain remote modules.'
+    required: false
```

Comment:
Note that I have added inputs for ITK and external module CMake inputs in an attempt to make this action extensible for external modules with ITK remote module dependencies. I.e. an external module with a dependency on ITKMeshToPolyData could build ITK with `-DModule_MeshToPolyData:BOOL=ON`, etc.